app/javascript/selectors/answersSelector.js

```diff
@@ -1,10 +1,43 @@
 import {
   getAllActiveQuestions,
   getAllActiveQuestionsForCategory,
-  getAllQuestionsByCategoryId
+  getAllActiveQuestionIdsForCategory,
 } from "./questionsSelector";
 import { getCategoriesForBuilding } from "./categoriesSelector";
 
+export function getAnswersForBuilding(buildingId, state) {
+  let answers = state.buildings[buildingId].answers;
+  answers = Object.keys(answers).filter((id) => {
+    let email = answers[id].delegation_email;
+    let text = answers[id].text;
+    return !email && !text;
+  })
+  .reduce((newAnswers, id) => {
+    newAnswers[id] = answers[id];
+    return newAnswers
+  }, {});
+  return answers
+}
+
+export function getAnswersForCategoryAndBuilding(categoryId, buildingId, state) {
+  const buildingQuestions = state.buildings[buildingId].questions;
+  let categoryQuestions = state.categories[categoryId].questions;
+  categoryQuestions = buildingQuestions.filter((id) => {
```

Comment:
Hmm I'm not sure this filter is necessary...doesn't `categoryQuestions` already have the question ids we care about? This gives us all questions in the building that are part of this specific category, but `categoryQuestions` is being used later just to check whether an answer in this building belongs to the category